File: src/test/java/org/jesperancinha/airports/containers/AirportContainerTest.java
package org.jesperancinha.airports.containers;

import org.jesperancinha.airports.pojos.Airport;
import org.junit.Test;

import java.util.List;

import static org.hamcrest.Matchers.hasSize;
import static org.hamcrest.Matchers.notNullValue;
import static org.junit.Assert.assertThat;

/**
 * Created by joaofilipesabinoesperancinha on 08-08-16.
 */
public class AirportContainerTest {
    @Test
    public void getAirports() throws Exception {
        final AirportContainer airportContainer = new AirportContainer();

        List<Airport> airports = airportContainer.getAirports();
        Airport airport = airports.get(0);

        assertThat(airports, hasSize(46506));
        assertThat(airport.getId(), notNullValue());
    }
}

File: src/test/kotlin/org/jesperancinha/airports/containers/AirportContainerTest.kt
package org.jesperancinha.airports.containers

import org.hamcrest.Matchers
import org.jesperancinha.airports.pojos.Airport
import org.junit.Assert
import org.junit.Test

/**
 * Created by joaofilipesabinoesperancinha on 08-08-16.
 */
class AirportContainerTest {
    @Test
    fun `should run airport test container`() {
        val airportContainer = AirportContainer()
        val airports: List<Airport> = airportContainer.airports
        val airport = airports[0]
        Assert.assertThat(airports, Matchers.hasSize(46506))
        Assert.assertThat(airport.id, Matchers.notNullValue())
    }
}